es-models/gene_centric/gene_centric.mapping.yaml

```diff
@@ -2,7 +2,13 @@ _size:
   enabled: true
 _source:
   excludes:
-  - case.*
+  - case.*.*
+  - case.consent_type
+  - case.d*
+  - case.i*
+  - case.p*
+  - case.s*
+  - case.*_*_*
```

Comment:
Would it make sense to comment these (and possibly the excludes section in case_centric) to provide additional details?

Reply:
Sadly pre-commit strips off all comments from the yaml file :(

Reply:
That makes sense. Thank you!